### PR TITLE
fix: Add github env.

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

File: src/environments/github/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# GitLab Environments
+
+> The GitLab focus environments.
```
Comment (douglasduteil) [inline src/environments/github/README.md:3]:
```suggestion
# GitHub Environments

> The GitHub focus environments.
```

**Review by douglasduteil — COMMENTED** [review]
Looks promising :)

**Review by douglasduteil — APPROVED** [review]
Go merge

**Comment by douglasduteil** [comment]
## BREAKING CHANGE: We should now use a common ci environments

```diff
- import gitlab from "@socialgouv/kosko-charts/environments/gitlab";
+ import gitlab from "@socialgouv/kosko-charts/environments";

- const envParams = gitlab(process.env);
+ const envParams = environments(process.env);
```

File: src/environments/github/README.md (hunk 1 above)
Comment (gary-van-woerkens) [reply src/environments/github/README.md]: Done

**Pull request merged**